lib/public_key/src/public_key.erl

```diff
@@ -1228,6 +1232,41 @@ pkix_test_data(#{} = Chain) ->
 pkix_test_root_cert(Name, Opts) ->
     pubkey_cert:root_cert(Name, Opts).
 
+%%--------------------------------------------------------------------
+-spec ocsp_status(binary(), list(), undefined | binary()) ->
```

Comment:
We need to better know the return value of this function as it is an API function. It looks to me like it returns {ok, [#'SingleResponse'{}]  | {error, Reason}   where we need to specify reason too.

Reply:
> We need to better know the return value of this function as it is an API function. It looks to me like it returns {ok, [#'SingleResponse'{}] | {error, Reason} where we need to specify reason too.

done